web/webpack.config.ts

```diff
@@ -218,6 +221,11 @@ const config = (
                 chunks: ["error-styles"],
                 publicPath: production ? "/static/webpack-bundles/" : "/webpack/",
             }),
+            new CompressionPlugin<ZopfliOptions>({
+                // Use zopfli to write pre-compressed versions of text files
+                test: /\.(js|css|html)$/,
+                algorithm: gzip,
+            }),
```

Comment:
Just copying from the chat.zulip.org thread that we probably do need to do something to make this not run in development? I at least see nothing obvious in the documentation or otherwise that would make it not.

Reply:
I verified that this would still run with `./tools/webpack --watch --port=...`. Added a commit which makes the plugin conditional on production.